prince-of-versions/src/main/java/co/infinum/princeofversions/mvp/presenter/impl/PovPresenterImpl.java

```diff
@@ -92,4 +111,90 @@ public void onError(@ErrorCode int error) {
             }
         });
     }
+
+    private boolean checkIfUpdateShouldBeMade(VersionContext version) {
```

Comment:
I saw your comment for the PR. I agree that this is too complicated and it does nothing.  What was the idea behind it? Why could not you just check if sdk is supported?